docker-compose.yml

```
@@ -14,9 +15,9 @@ services:
       MYSQL_ROOT_PASSWORD: password
       MYSQL_USER: user
       MYSQL_PASWORD: password
-      ports:
-        - 13306:13306
-      volumes:
-        - ./mysql/data:/var/lib/mysql
-        - ./mysql/my.conf:/etc/mysql/conf.d/my.cnf
-        - ./mysql/init_db:/docker-entrypoint-initdb.d
+    ports:
+      - 13306:13306
+    volumes:
+      - ./mysql/data:/var/lib/mysql
```

開発するだけだから名前付きvolumeに変えて権限周りを回避してもいいかも

```suggestion
      - mysql-data:/var/lib/mysql
```

https://qiita.com/ysd_marrrr/items/e8a50c43cff87951385c#%E8%A7%A3%E6%B1%BA%E6%B3%951-%E3%83%87%E3%83%BC%E3%82%BF%E3%81%AE%E6%B0%B8%E7%B6%9A%E5%8C%96%E3%81%AF%E5%90%8D%E5%89%8D%E4%BB%98%E3%81%8D%E3%83%9C%E3%83%AA%E3%83%A5%E3%83%BC%E3%83%A0%E3%81%AB%E3%81%99%E3%82%8B